yb-voyager/src/srcdb/source.go

```diff
@@ -202,6 +203,26 @@ type Ora2pgConfig struct {
 	ModifyStruct     string
 }
 
+type PgDumpArgs struct {
```

Comment:
For the boolean ones, should be boolean type no?

Reply:
`ini` parser read it in form of strings only. We will need to parse it into respective datatype first, but in the end we just want pg_dump args in the form of string.